src/libraries/System.Drawing.Common/tests/Drawing2D/HatchBrushTests.cs

```diff
@@ -54,6 +54,7 @@ public void Ctor_HatchStyle_ForeColor_BackColor(HatchStyle hatchStyle, Color for
         [Theory]
         [InlineData(HatchStyle.Horizontal -1 )]
         [InlineData(HatchStyle.SolidDiamond + 1)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/56048", TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.MacCatalyst)]
```

Comment:
@MaximLipnin you can use `[ConditionalTheory(Helpers.IsDrawingSupported)]` instead, that will disable the test on platforms where System.Drawing.Common is not supported

Reply:
@akoeplinger Is it for all the System.Drawing.Common tests which I've disabled or just for the two you mentioned?

Reply:
for all

Reply:
done, thanks

Reply:
weird, the System.Drawing.Common tests keep failing one per run. Perhaps, it makes sense to use ` [ConditionalClass(Helpers.IsDrawingSupported)]`